<ui version="4.0" >
 <class>historySettingsClass</class>
 <widget class="QWidget" name="historySettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>historySettings</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="saveHistoryBox" >
        <property name="text" >
         <string>Save message history</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="nilHistoryBox" >
        <property name="text" >
         <string>Save message history for " Not in list" contacts</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="serviceBox" >
        <property name="text" >
         <string>Save service and blocked users messages</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="onlineNotifyBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Save online notifications</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QCheckBox" name="offlineNotifyBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Save offline notifications</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QCheckBox" name="readAwayNotifyBox" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Save read-away notifications</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="recentBox" >
          <property name="text" >
           <string>Show recent messages in messaging window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="messagesCountBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>10</number>
          </property>
          <property name="value" >
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>recentBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>messagesCountBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>247</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel" >
     <x>378</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>onlineNotifyBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>199</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel" >
     <x>150</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>offlineNotifyBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>78</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel" >
     <x>86</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serviceBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>readAwayNotifyBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>241</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel" >
     <x>205</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
